modules/apps/object/object-test/src/testFunctional/macros/ObjectField.macro

```diff
@@ -67,6 +67,12 @@ definition {
 		Click(locator1 = "ObjectField#MODAL_CLOSE_BUTTON");
 	}
 
+	macro insertScriptOnFormulaField {
+		Type(
+			locator1 = "ObjectField#SCRIPT_ON_FORMULA_FIELD",
```

Comment:
Instead of creating a new path, you could use an generic path for this issue. For example, [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro#L915):
```
Type(
	key_text = "External Reference Code",
	locator1 = "TextInput#ANY",
	value1 = ${externalReferenceCode});
```